packages/compass-database/src/components/database.tsx

```diff
@@ -12,7 +10,11 @@ const databaseStyles = css({
   height: '100%',
 });
 
-function Database() {
+export interface DatabaseProps {
+  logger: LoggerAndTelemetry;
```

Comment:
This can be just calling useLogger hook inside the component and then you don't need to pass it as a prop, right?

Reply:
Yeah, I have the memory of a goldfish and didn’t remember that we literally have that for this purpose. :slightly_smiling_face: Fixed!